src/client/pages/OfferNew/FaqSection.tsx

```diff
@@ -151,33 +138,99 @@ export const Accordion: React.FC<AccordionProps> = ({ headline, body }) => {
   )
 }
 
-export const FaqSection: React.FC = () => {
-  const pathLocale = useCurrentLocale()
-  const language = getIsoLocale(pathLocale)
-  const faqs = useFaqsQuery({ variables: { language } })
-  const languageData = faqs?.data?.languages[0]
-  const textKeys = useTextKeys()
+type FaqPanelProps = {
+  list?: AccordionProps[]
+}
 
+const FaqPanel = ({ list }: FaqPanelProps) => {
   return (
     <>
-      <SectionWrapper>
-        <Container>
-          <Column>
-            <HeadingWhite>{textKeys.OFFER_FAQ_HEADING()}</HeadingWhite>
-            <AccordionsWrapper>
-              {languageData?.faqs?.map((faq) => (
-                <Accordion
-                  key={faq?.id}
-                  id={faq?.id}
-                  headline={faq!.headline!}
-                  body={faq!.body!}
-                />
-              ))}
-            </AccordionsWrapper>
-          </Column>
-          <ColumnSpacing />
-        </Container>
-      </SectionWrapper>
+      {list?.map((listItem) => (
+        <Accordion
+          key={listItem?.headline}
+          headline={listItem!.headline}
+          body={listItem!.body!}
+        />
+      ))}
     </>
   )
 }
+
+type FaqSectionProps = {
+  variants: QuoteBundleVariant[]
+}
+
+export const FaqSection = ({ variants }: FaqSectionProps) => {
+  const textKeys = useTextKeys()
+
+  const quotes = variants
+    .map((v) => v.bundle.quotes)
+    .reduce((accumulated, quotes) => [...accumulated, ...quotes], [])
+
+  const { isoLocale } = useCurrentLocale()
+
+  const { data, loading } = useGetContractFaqsQuery({
+    variables: {
+      input: {
+        contractTypes: quotes.map((q) => q.typeOfContract),
+        locale: isoLocale,
+      },
+    },
+  })
+
+  const tabItems = React.useMemo(() => {
+    if (!data || !data.contractFaqs?.faqs?.length) return null
+
+    return data.contractFaqs.faqs.map((d) => ({
+      id: d.title || '',
+      // if only one FAQ: show no tab (empty string for 'name')
+      name: data.contractFaqs?.faqs?.length ?? 0 < 2 ? '' : d.title!,
```

Comment:
What is happening here? 😅  maybe change to:
```suggestion
      name: (data.contractFaqs?.faqs?.length ?? 0) < 2 ? '' : d.title!,
```

Reply:
Isn't it crystal clear :D ? 

I agree it's messy!